src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -124,6 +124,33 @@ def put(self, request: Request, project, rule) -> Response:
         if serializer.is_valid():
             data = serializer.validated_data
 
+            explicit_opt_out = request.data.get("optOutExplicit")
+            edit_opt_out = request.data.get("optOutEdit")
+            if explicit_opt_out or edit_opt_out:
+                neglected_rule = NeglectedRule.objects.filter(
```

Comment:
would there ever be more than 1 object returned here? is that possible to occur?

Reply:
I think there should only ever be 1

Reply:
okay -- could be worth adding a comment to signify that, or changing it to use `.get` and logging an error/ warning if more than 1 is returned